modules/core/src/main/scala/org/scalasteward/core/nurture/NurtureAlg.scala

```diff
@@ -80,6 +84,11 @@ final class NurtureAlg[F[_]](
       }
     } yield ()
 
+  def getNonSbtUpdates(repo: Repo): F[List[Update.Single]] =
+    for {
+      maybeScalafmt <- scalafmtAlg.getScalafmtUpdate(repo)
+    } yield List(maybeScalafmt).flatten
```

Comment:
```suggestion
    } yield maybeScalafmt.toList
```

Reply:
This is intended so non-sbt update can be added like this.
```scala
for {
  maybeScalafmt <- scalafmtAlg.getScalafmtUpdate(repo)
  maybeFoo <- fooAlg.getUpdate()
} yield List(maybeScalafmt, maybeFoo).flatten
```